apps/mobile/package.json

```diff
@@ -97,7 +97,7 @@
     "react-test-renderer": "18.2.0",
     "ts-jest": "29.1.4",
     "tslib": "2.6.2",
-    "typescript": "5.3.3"
+    "typescript": "5.4.5"
```

Comment:
Good job updating these. Kyran mentioned to me previously about setting up `syncpack` to keep these in check. I opened an issue for it [here](https://github.com/leather-wallet/mono/issues/226) 

That could help us keep versions of certain packages the same throughout the monorepo

Reply:
Edgar already set up syncpack, and we have a CI job for it? 🤔

Reply:
OK. I guess I checked for it in the `extension` by mistake and don't understand how it works. I thought it should help us avoid having to update all the dependancies in each package.json?